.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,8 +0,0 @@
-Checklist
```

Comment:
Let's put this file back. The rerender removed it.

Reply:
Fixed in 252bd6a.